src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/DockerProcessor.kt

```diff
@@ -52,8 +57,7 @@ class DockerProcessor(private val docker: DockerClient) {
             val compiledRmd = FilenameUtils.removeExtension(fileName) + PDF_EXTENSION
             return Paths.get(parentDir).resolve(compiledRmd).toFile()
         } catch (e: Exception) {
-            // TODO : log it after merging logging PR
-            e.printStackTrace()
+            LOGGER.error("failed to compile file: {}", e)
```

Comment:
If you just want to add stack trace to the log message, you do not need a placeholder. If you meant to add filename to the message, please provide it in the arguments.

Reply:
Done